### PR TITLE
Update fixture library for SVI dashboard

### PR DESCRIPTION
**Pull request by andersonad**
Note: I don't have any SVI fields locally, so although this passes the tests of loading into kibana's index, I could not confirm whether the dashboard/fixtures were working correctly.

Please make sure to do so before pushing to everyone.

### CODE REVIEW

File: src/templates/visualization/common-addresses-svi-chars.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      "uiStateJSON": "{\"vis\":{\"params\":{\"sort\":{\"columnIndex\":0,\"direction\":null}}}}",
+      "description": "",
+      "version": 1,
+      "kibanaSavedObjectMeta": {
+        "searchSourceJSON": "{\"index\":\"93345-va-richmond_fire_and_emergency_services-fire-incident*\",\"query\":{\"query\":{\"match_all\":{}},\"language\":\"lucene\"},\"filter\":[{\"meta\":{\"index\":\"93345-va-richmond_fire_and_emergency_services-fire-incident*\",\"negate\":true,\"type\":\"phrase\",\"key\":\"address.address_line1\",\"value\":\"\",\"params\":{\"query\":\"\",\"type\":\"phrase\"},\"disabled\":false,\"alias\":null},\"query\":{\"match\":{\"address.address_line1\":{\"query\":\"\",\"type\":\"phrase\"}}},\"$state\":{\"store\":\"appState\"}}]}"
```
Comment (chopchop505) [inline src/templates/visualization/common-addresses-svi-chars.json]: all references to richmond need to be removed

**Comment by chopchop505** [comment]
image

**Comment by chopchop505** [comment]
So if we want to use these region maps - we need to simplify the geometries, and upload each states GeoJSON file or find one that covers the entire US. 

I got Virginias from here: https://github.com/arcee123/GIS_GEOJSON_CENSUS_TRACTS.  But even that is too big, 50MB+.  It should really be less than 5MB.

**Comment by chopchop505** [comment]
Here is the guide for creating a new layer: https://github.com/elastic/ems-file-service

**Comment by andersonad** [comment]
If I have a full country .json, what would the max size be?

**Comment by andersonad** [comment]
Entire nation, 25% simplification (still pretty good resolution on shapes) 
[SVI2016_US.zip](https://github.com/StatEngine/se-fixtures/files/3167959/SVI2016_US.zip)
topojson file

**Comment by andersonad** [comment]
17.6MB option with less info (but we may never use that info)
[SVI2016_US-no-location.zip](https://github.com/StatEngine/se-fixtures/files/3168021/SVI2016_US-no-location.zip)

**Comment by chopchop505** [comment]
The visualizations are loading, but not tied into dashboard correctly

image

File: src/templates/dashboard/svi-introduction.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "_id": "dashboard:svi-introduction",
+  "_type": "doc",
+  "_source": {
+    "type": "dashboard",
+    "dashboard": {
+      "title": "SVI Introduction",
+      "hits": 0,
+      "description": "Introduces some visualizations featuring the Social Vulnerability Index (SVI) dataset, which is based primarily on the American Community Survey (ACS).",
```
Comment (chopchop505) [inline src/templates/dashboard/svi-introduction.json]: cd80b170-b130-11e8-b5da-711bafd57368 should be a human readable id (i.e visualization:blah

Comment (chopchop505) [inline src/templates/dashboard/svi-introduction.json]: Kibana strips "visualization" out, so "visualization:median-response-time-by-disabilities-and-elderly" should just be "median-response-time-by-disabilities-and-elderly"

Comment (chopchop505) [reply src/templates/dashboard/svi-introduction.json]: Update all the references

File: src/templates/visualization/hour-of-the-day-heatmap.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
...
...{\"input\":{\"query\":{\"query_string\":{\"query\":\"description.hour_of_day: 13\"}}},\"label\":\"13:00\"},{\"input\":{\"query\":{\"query_string\":{\"query\":\"description.hour_of_day: 14\"}}},\"label\":\"14:00\"},{\"input\":{\"query\":{\"query_string\":{\"query\":\"description.hour_of_day: 15\"}}},\"label\":\"15:00\"},{\"input\":{\"query\":{\"query_string\":{\"query\":\"description.hour_of_day: 16\"}}},\"label\":\"16:00\"},{\"input\":{\"query\":{\"query_string\":{\"query\":\"description.hour_of_day: 17\"}}},\"label\":\"17:00\"},{\"input\":{\"query\":{\"query_string\":{\"query\":\"description.hour_of_day: 18\"}}},\"label\":\"18:00\"},{\"input\":{\"query\":{\"query_string\":{\"query\":\"description.hour_of_day: 19\"}}},\"label\":\"19:00\"},{\"input\":{\"query\":{\"query_string\":{\"query\":\"description.hour_of_day: 20\"}}},\"label\":\"20:00\"},{\"input\":{\"query\":{\"query_string\":{\"query\":\"description.hour_of_day: 21\"}}},\"label\":\"21:00\"},{\"input\":{\"query\":{\"query_string\":{\"query\":\"description.hour_of_day: 22\"}}},\"label\":\"22:00\"},{\"input\":{\"query\":{\"query_string\":{\"query\":\"description.hour_of_day: 23\"}}},\"label\":\"23:00\"}]}}],\"listeners\":{}}",
```
Comment (chopchop505) [inline src/templates/visualization/hour-of-the-day-heatmap.json]: image

Comment (chopchop505) [reply src/templates/visualization/hour-of-the-day-heatmap.json]: I'm still seeing this when running.  Y axis should be reversed.  X labels (0, 1, 2, 3, 4...) should be removed

**Comment by chopchop505** [comment]
More bad news: we need a geojson file.
```
Each layer object points to an external vector file that contains a geojson FeatureCollection. The file must use the WGS84 coordinate reference system and only include polygons
```

**Comment by chopchop505** [comment]
https://www.elastic.co/guide/en/kibana/current/settings.html#regionmap-settings

**Comment by andersonad** [comment]
It seems elasticsearch map service should support topojson for region maps?
https://github.com/elastic/kibana/issues/14331

File: src/templates/dashboard/svi-introduction.json (hunk 2 above)
Comment (andersonad) [reply src/templates/dashboard/svi-introduction.json]: This visualization is just the already pre-existing "Response Map" visualization (that is already in the fixture library). Does Kibana recode things on export? (Making the change though)

**Comment by andersonad** [comment]
ok, If not, 2 more options:
Here is a geojson of the nation with geometry simplified to 25 percent, uncompressed size is 42 MB.
[SVI2016_US_one_file.zip](https://github.com/StatEngine/se-fixtures/files/3188299/SVI2016_US_one_file.zip)

**alternatively**: I have a zipfile with each individual state extracted as a geojson. The disadvantage here (perhaps) is that if we someday get some cities on the state borders (e.g. Kansas City Missouri), there might be problems with displaying that in Kibana. Otherwise, largest individual state file size is 8.8 MB (CA). Most states are under 3 MB. I can't paste it here because the zip file is 27 MB.

**Comment by chopchop505** [comment]
Hmm. I was having troules with loading the topojson.  Can you try adding the file to your dev instance and see if it loads. 
https://www.elastic.co/blog/custom-region-maps-in-kibana-6-0

**Comment by andersonad** [comment]
As a note, we decided to try out individual geojson files corresponding to each state.

**Comment by andersonad** [comment]
It just occurred to me that since we made that decision, those region maps cannot be automatically deployed and "just work" because we don't a priori know the state, right?

**Comment by chopchop505** [comment]
So remaining items:
- [ ] Visualizations are loading, but not tied into dashboard correctly (See previous comment)
- [ ] Heatmap (see previous comment) 
- [ ] State is passed in as a local variable, {{ FireDepartment.state }}.  You should be able to programmatically generate the region map based on this.

Once complete, run locally and verify new dashboards load correctly

**Comment by andersonad** [comment]
Ok, branch re-updated to reflect backend changes to region maps and to correct an error in the response-map visualization on the dashboard (I was using responses rather than incidents). everything should load correctly now after you pull the new (replaced) commit.

**Pull request merged**